src/Compilers/CSharp/Test/Semantic/Semantics/ObjectAndCollectionInitializerTests.cs

```diff
@@ -912,6 +912,8 @@ public static void Main()
             Left: IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid) (Syntax: 'x')
                 Children(1):
                     IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'x')
+                      Children(1):
+                          IInstanceReferenceExpression (OperationKind.InstanceReferenceExpression, Type: System.Int32) (Syntax: 'int')
```

Comment:
>IInstanceReferenceExpression (OperationKind.InstanceReferenceExpression, Type: System.Int32) (Syntax: 'int') [](start = 26, length = 108)

Should this node be marked as implicit?

Reply:
it is implicit, but test result output doesnt print whether something is implicit or not.

Reply:
which should be fixed as part of this change https://github.com/dotnet/roslyn/issues/21330